azimuth_caas_operator/utils/ansible_runner.py

```diff
@@ -364,8 +456,8 @@ async def _get_ansible_runner_events(client, job_name, namespace):
     if len(pod_names) == 0 or len(pod_names) > 1:
         # TODO(johngarbutt) only works because our jobs don't retry,
         # and we don't yet check the pod is running or finished
-        LOG.debug(f"Found pods:{pod_names} for job {job_name} in {namespace}")
-        return
+        LOG.debug(f"Found pods: {pod_names} for job {job_name} in {namespace}")
+        return []
```

Comment:
NIT: this should really be a separate PR with a nice commit message about what if fixes.

Reply:
Done